.github/workflows/bump-deps.yaml

```diff
@@ -0,0 +1,88 @@
+# Description: Run this by visiting https://github.com/brave/brave-core/actions/workflows/bump-deps.yaml
+# and clicking the `Run workflow` button on the top-right.
+# You can also run it from cli by: `gh workflow run bump-deps.yaml [-f dep=vendor/web-discovery-project] [-f ref=commit-hash]`
+# If `ref` is not given, we will try to use the latest commmit (tip) of a branch called `master` or `main`
+name: Bump DEPS
```

Comment:
Let's rename (also the file) to `Bump dependency` if it's about a single one at once.

Reply:
`Bump/update dependency [ref] in DEPS?`

Reply:
[91976e1](https://github.com/brave/brave-core/pull/16068/commits/91976e1e8cf883a6ae96ed57355971f59fdf02a8)